04/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="keywords" content="receta picadillo, ingredientes, picadillo, platillo mexicano">
+  <meta name="description" content="Ingredientes secretos y fórmula exacta para preparar Picadillo">
+  <meta name="auhtor" content="Abril Muñoz">
+  <title>Receta de Picadillo</title>
+</head>
+<body>
+  <main>
+    <header>
+      <p>
+        <h1>Picadillo</h1>
+      </p>
+    </header>
+    <article>
+      <h2>Ingredientes</h2>
+    <ul>
+      <li>1 Cucharada de aceite vegetal</li>
+      <li>1/4 De pieza de cebolla picada finamente</li>
+      <li>800 Gramos de carne molida de res</li>
+      <li>5 Jitomate cortados en cuartos</li>
+      <li>1 Diente de ajo</li>
+      <li>2 Cubos de Concentrado de Tomate con Pollo CONSOMATE®</li>
+      <li>2 Chiles chipotle adobados</li>
+      <li>2 Papas cortadas en cubos y cocidas</li>
+      <li>2 Zanahorias cortadas en cubos y cocidas</li>
+      <li>1/2 Taza de chícharos cocidos</li>
+      <li>3 Hojas de laurel</li>
+    </ul>
+    </article>
+    <section>
+      <h2>Instrucciones para cocinar</h2>
+    <ol>
+      <li>Calienta el aceite, fríe la cebolla hasta que esté transparente</li>
+      <li>Agrega la carne y cocina por 5 minutos moviendo ocasionalmente</li>
+      <li>Licúa los jitomates con el ajo, el Concentrado de Tomate con Pollo CONSOMATE® y los chiles</li>
+      <li>Vierte sobre la carne, tapa y cocina hasta que comience a hervir.</li>
+      <li>Agrega las papas, las zanahorias, los chícharos y el laurel</li>
+      <li>Cocina por 5 minutos más y ofrece</li>
```

Comment:
If every step finishes without or with a dot, all steps should finish the same way